share/spice/caniuse_lists/caniuse_lists.js

```diff
@@ -0,0 +1,164 @@
+(function (env) {
+    'use strict';
+
+    env.ddg_spice_caniuse_lists = function(api_result) {
+        
+        if (!api_result) {
+            return Spice.failed('caniuse_lists');
+        }
+
+        //parse the query for name of feature to be shown
+        var query  = DDG.get_query(),
+            result = query.match(/css[2-3]?|html5|svg|js api/ig),
+
+            //get relevant array from the returned JSON object
+            statuses      = api_result['statuses'],
+            data          = api_result['data'],
+            browsers      = api_result['agents'],
+            supported     = {'ie': true,'firefox': true,'chrome': true,'safari': true},
+            required_data = [];
+
+        //pick only the required features based on category
+        for( var feature in data ) {
+
+            var obj = data[feature];
+            for(var search_term in result) {
+                if( obj['categories'].indexOf(result[search_term].toUpperCase()) != -1) {
+                    required_data.push(obj);
+                    break;
+                }
+            }
+        }
+
+        //sort by usage percentage, since there are many features
+        required_data.sort(function(a,b) {
+            return b.usage_perc_y - a.usage_perc_y;
+        });
+        
+        // Render the response
+        Spice.add({
+            id: 'caniuse_lists',
+            name: 'Can I Use',
+            data: required_data,
+            meta: {
+                sourceName: 'caniuse.com',
+                sourceUrl: 'http://caniuse.com/'
+            },
+            normalize: function(item) {
+                supported = {'ie': true,'firefox': true,'chrome': true,'safari': true};
```

Comment:
we need to reset the values to `true` for every tile, I removed the above initialisation now